$MirOS: ports/devel/autoconf/2.63/patches/patch-lib_m4sugar_m4sugar_m4,v 1.1 2009/07/15 16:43:37 bsiegert Exp $
$OpenBSD: patch-lib_m4sugar_m4sugar_m4,v 1.1.1.1 2008/08/19 23:52:19 espie Exp $
--- lib/m4sugar/m4sugar.m4.orig	Tue Aug 26 12:30:59 2008
+++ lib/m4sugar/m4sugar.m4	Mon Aug 10 17:22:09 2009
@@ -1708,6 +1708,7 @@ m4_define([m4_expand_once],
 m4_define([m4_provide],
 [m4_define([m4_provide($1)])])
 
+m4_provide([AM_SANITY_CHECK])
 
 # m4_provide_if(MACRO-NAME, IF-PROVIDED, IF-NOT-PROVIDED)
 # -------------------------------------------------------
@@ -2392,13 +2393,17 @@ m4_define([m4_version_compare],
 # file is optional, provided by Autoconf but absent in Bison.
 m4_sinclude([m4sugar/version.m4])
 
+# m4_PACKAGE_VERSIONNR
+# --------------------
+m4_define([m4_PACKAGE_VERSIONNR],
+	m4_bpatsubst(m4_defn([m4_PACKAGE_VERSION]), [-.*], []))
 
 # m4_version_prereq(VERSION, [IF-OK], [IF-NOT = FAIL])
 # ----------------------------------------------------
 # Check this Autoconf version against VERSION.
 m4_define([m4_version_prereq],
 m4_ifdef([m4_PACKAGE_VERSION],
-[[m4_if(m4_version_compare(]m4_dquote(m4_defn([m4_PACKAGE_VERSION]))[, [$1]),
+[[m4_if(m4_version_compare(]m4_dquote(m4_defn([m4_PACKAGE_VERSIONNR]))[, [$1]),
 	[-1],
 	[m4_default([$3],
 		    [m4_fatal([Autoconf version $1 or higher is required],
